js/scripts.js

```diff
@@ -0,0 +1,14 @@
+var a = '-2';
+    b = '4';
```

Comment:
masz świadomość, że nie deklarujesz tutaj zmiennej `b` tylko przypisujesz jej nową wartość (ona nie istnieje, więc js stworzy ją za Ciebie) -bądź tego świadomy bo potencjalnie prowadzi to do problemów w przyszłości.